### DISCUSSION TITLE
Test Certification (PU/SMC-B)

### DISCUSSION THREAD

**Discussion by dirkexe** [Digitaler-Impfnachweis/certification-apis#132] [Q&A] [answered]
What are the requirements for a medical practice to generate a **test certificate**? Are medical practices that operate as test centers allowed to use the Issuer API via the Authentication with TI SMC-B?

The Issuer API currently returns:

```
400 Bad Request
Sorry, there is something invalid in your request: Test and recoveries certificates are not allowed.
```

**Comment by dirkexe** [comment]
Kann ich etwas tun, um die Antwort auf meine Frage zu erhalten? Die Fehlermeldung in sich ist doch bereits falsch, da "recoveries certificates" erlaubt sind.

**Comment by thinkberg** [comment]
Es liegt an backend. Die Test Stage ist auch eine Art Drehkreuz für das Produktionssystem und bei fixes wird dort u.U. Der Stand zurückgedreht. Wir werden das ändern, und ab demnächst die QA Umgebung als Test Stage vorsehen. Dort ist der Zustand stabiler.

**Reply by thinkberg** [reply]
Sollte wieder gehen.

**Reply by dirkexe** [reply]
Die Meldung ist jetzt 400 "Sorry, there is something invalid in your request: Test certificates not allowed." Das ist jetzt logisch. Meine eigentliche Frage ist jedoch nicht damit beantwortet:

Darf eine Arztpraxis Test-Zerfifikate überhaupt über den Weg SMC-B ausstellen, wenn die Praxis als Testzentrum arbeitet? Oder ist und wird dieser Weg nicht vorgesehen?

**Comment by oliver-steinbrecher** [comment, marked as answer]
Aktuell ist es nicht vorgesehen, dass man als Arzt Testzertifikate erstellen kann.

**Reply by msteffenDev** [reply]
Soll aber laut diverser Rückmeldungen aus unserer Kundschaft ab August 2021 möglich sein, sofern ich den Sachverhalt richtig verstanden habe. Zumindestens werden nur dann die Tests auch vergütet...
